imageboard/ImageBoardNodeProj.njsproj

```diff
@@ -0,0 +1,107 @@
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'" />
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'" />
+  <ItemGroup>
+    <Content Include="package.json" />
+    <Content Include="README.md" />
+    <TypeScriptCompile Include="app.ts" />
+    <TypeScriptCompile Include="db.ts" />
+    <TypeScriptCompile Include="routes\index.ts" />
+    <Content Include="views\board.jade" />
+    <Content Include="views\image.jade" />
+    <Content Include="views\index.jade" />
+    <Content Include="views\layout.jade" />
+    <Content Include="views\newboard.jade" />
+    <Content Include="views\newpin.jade" />
+    <Content Include="views\user.jade" />
+    <Content Include="dump\mydb\images.metadata.json" />
+    <Content Include="dump\mydb\users.metadata.json" />
+    <Compile Include="public\javascripts\fb.js" />
+    <Content Include="public\stylesheets\style.css" />
+    <TypeScriptCompile Include="typings\express\express-middleware.d.ts" />
+    <TypeScriptCompile Include="typings\express\express.d.ts" />
+    <TypeScriptCompile Include="typings\mongodb\mongodb.d.ts" />
+    <TypeScriptCompile Include="typings\node\node.d.ts" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="bin" />
+    <Folder Include="bin\Debug" />
+    <Folder Include="dump" />
+    <Folder Include="dump\mydb" />
+    <Folder Include="obj" />
+    <Folder Include="obj\Debug" />
+    <Folder Include="obj\Debug\TempPE" />
+    <Folder Include="public" />
+    <Folder Include="public\images" />
+    <Folder Include="public\javascripts" />
+    <Folder Include="public\stylesheets" />
+    <Folder Include="routes" />
+    <Folder Include="typings" />
+    <Folder Include="typings\express" />
+    <Folder Include="typings\mongodb" />
+    <Folder Include="typings\node" />
+    <Folder Include="views" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <!--Do not delete the following Import Project.  While this appears to do nothing it is a marker for setting TypeScript properties before our import that depends on them.-->
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.targets" Condition="False" />
+  <Import Project="$(VSToolsPath)\Node.js Tools\Microsoft.NodejsTools.targets" />
+  <ProjectExtensions>
+    <VisualStudio>
+      <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
+        <WebProjectProperties>
+          <UseIIS>False</UseIIS>
+          <AutoAssignPort>True</AutoAssignPort>
+          <DevelopmentServerPort>0</DevelopmentServerPort>
+          <DevelopmentServerVPath>/</DevelopmentServerVPath>
+          <IISUrl>http://localhost:48022/</IISUrl>
+          <NTLMAuthentication>False</NTLMAuthentication>
+          <UseCustomServer>True</UseCustomServer>
+          <CustomServerUrl>http://localhost:1337</CustomServerUrl>
```

Comment:
Does this port actually have any effect? Does it make VS launch a browser with `http://localhost:1337`? Because in app.ts we just use port 3000.